# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWaylandDisplay.h
# Commit: 4c4c96ad829d
# Full Hash: 4c4c96ad829d55b88ee52a320cab335ac2b86340
# Author: stransky <stransky@redhat.com>
# Date: 2025-02-05 16:56:59
# Regressor Bug: 1944490
# File Overlap Count: 1
# Description:
#   Bug 1944490 [Linux] Ensure DMABufFormats provides basic dmabuf formats r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D236688
# ==============================================================================

diff -r 25e327e7c9ad -r 4c4c96ad829d widget/gtk/nsWaylandDisplay.h
--- a/widget/gtk/nsWaylandDisplay.h	Tue Feb 04 20:14:58 2025 +0000
+++ b/widget/gtk/nsWaylandDisplay.h	Tue Feb 04 20:44:48 2025 +0000
@@ -108,6 +108,7 @@
   }
   RefPtr<DMABufFormats> GetDMABufFormats() const { return mFormats; }
   bool HasDMABufFeedback() const { return mDmabufIsFeedback; }
+  void EnsureDMABufFormats();
 
   ~nsWaylandDisplay();
 
